src/common/Mutex.cc

```diff
@@ -61,7 +61,7 @@ Mutex::Mutex(const std::string &n, bool r, bool ld,
       _register();
   }
   else {
-    // If the mutex type is PTHREAD_MUTEX_NORMAL, deadlock detection
+    // If the mutex type is PTHREAD_MUTEX_DEFAULT, deadlock detection
```

Comment:
In this comment, PTHREAD_MUTEX_NORMAL seems to be correct.

Please read: http://pubs.opengroup.org/onlinepubs/009695399/functions/pthread_mutex_lock.html.